src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,153 @@
 package core.basesyntax;
 
+//import java.util.HashMap;
+import java.util.Objects;
+import java.util.Random;
+
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    public double EXPANSION_COEFFICIENT = 2;
+    public double LOAD_FACTOR = 0.75;
+    private Node<K, V>[] values;
+    private static final int defaultCapacity = 16;
```

Comment:
Please rename this variable to be in CONSTANT_CASE. 
Ref: https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names